tools/syz-db/syz-db.go

```diff
@@ -205,8 +206,20 @@ func bench(target *prog.Target, file string) {
 		stats.Alloc>>20,
 		(stats.Mallocs-stats.Frees)>>20,
 		time.Since(start))
-	sink = corpus
-	_ = sink
+	n := len(corpus)
+	fmt.Printf("corpus size: %v\n", n)
+	if n == 0 {
+		return
+	}
+	sum := 0
+	lens := make([]int, n)
+	for i, p := range corpus {
+		sum += len(p.Calls)
+		lens[i] = len(p.Calls)
+	}
+	sort.Ints(lens)
+	fmt.Printf("program size: min=%v avg=%v max=%v 10%%=%v 50%%=%v 90%%=%v\n",
+		lens[0], lens[n/2], lens[n-1], lens[n/10], lens[n/2], lens[n*9/10])
```

Comment:
`lens[n/2]` is not avg, it's median. Should it have been `sum / n`?

Reply:
Right. Done.